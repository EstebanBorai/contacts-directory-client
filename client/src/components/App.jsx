import React from 'react';
import Header from 'components/Header';
import Contacts from 'components/Contacts';
import ContactsContext from 'contexts/ContactsContext';
import NavigationContext from 'contexts/NavigationContext';

const App = () => (
  <NavigationContext>
    <ContactsContext>
      <React.Fragment>
        <Header />
        <Contacts />
      </React.Fragment>
    </ContactsContext>
  </NavigationContext>
);

export default App;
