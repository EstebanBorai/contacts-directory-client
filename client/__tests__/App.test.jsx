import React from 'react';
import App from '../src/components/App';
import renderer from 'react-test-renderer';

test('should write "Hello"', () => {
  const component = renderer.create(<App />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
