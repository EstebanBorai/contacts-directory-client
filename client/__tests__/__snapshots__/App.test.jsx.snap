// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should write "Hello" 1`] = `
<div>
  Hello
</div>
`;
